src/org/zaproxy/zap/extension/frontendscanner/FrontEndScannerAPI.java

```diff
@@ -44,6 +66,44 @@ public String getPrefix() {
 
     @Override
     public String handleCallBack(HttpMessage msg) throws ApiException {
+        try {
+            String query = msg.getRequestHeader().getURI().getQuery();
```

Comment:
Ideally this should use `getEscapedQuery()` instead (to properly parse it later).